services/arv-git-httpd/git_handler_test.go

```diff
@@ -41,13 +41,14 @@ func (s *GitHandlerSuite) TestEnvVars(c *check.C) {
 }
 
 func (s *GitHandlerSuite) TestCGIError(c *check.C) {
+	log.Printf("git_handler_test: TestCGIError() - expecting a SplitHostPort error, so it is safe to ignore!")
```

Comment:
How about we just rename the test to TestCGIErrorOnSplitHostPortError and/or change "bogus" to "intentionally.broken.host.with.no.port"? I think we might go crazy if we start trying to explain away error messages that get logged during tests by adding more log messages to the tests...